repo/init.go

```diff
@@ -161,7 +161,7 @@ func addConfigExtensions(repoRoot string, testnet bool) error {
 		return err
 	}
 	var w WalletConfig = WalletConfig{
-		Type:             "spvwallet",
```

Comment:
We also do not want to initialize it with bitcoin cash by default here.

Reply:
Fixed